godot-core/Cargo.toml

```diff
@@ -11,12 +11,16 @@ categories = ["game-engines", "graphics"]
 default = []
 codegen-fmt = ["godot-ffi/codegen-fmt", "godot-codegen/codegen-fmt"]
 codegen-full = ["godot-codegen/codegen-full"]
-codegen-lazy-fptrs = ["godot-ffi/codegen-lazy-fptrs", "godot-codegen/codegen-lazy-fptrs"]
+codegen-lazy-fptrs = [
+    "godot-ffi/codegen-lazy-fptrs",
+    "godot-codegen/codegen-lazy-fptrs",
+]
```

Comment:
Was this formatting changed by some tool?

Asking because we don't usually do this (see other lines in this file) 🙂

Reply:
Out of curiosity, which one do you use? Not that my CLion TOML plugin un-formats it again 😬

Reply:
<https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml> this one